src/ResizeObserverController.ts

```diff
@@ -143,14 +149,38 @@ export default class ResizeObserverController {
 
     if (mutationObserverSupported) {
       this.mutationsObserver_ = new MutationObserver(this.refresh);
-
-      this.mutationsObserver_.observe(document, {
-        attributes: true,
-        childList: true,
-        characterData: true,
-        subtree: true
-      });
-    } else {
+      const options = {
+          attributes: true,
+          childList: true,
+          characterData: true,
+          subtree: true
+      };
+
+      this.mutationsObserver_.observe(document, options);
+
+      if (origAttachShadow) {
```

Comment:
looks good! how do the tests for this repo look? would be nice to have a few tests of this behavior

Reply:
Good question...will take a look.

Reply:
OK, updated the tests. PTAL!